shishito/library/modules/runtime/environment/appium_control_environment.py

```diff
@@ -0,0 +1,70 @@
+from appium import webdriver
+#from selenium import webdriver
+import time
+
+from shishito.library.modules.runtime.environment.shishito_environment import ShishitoEnvironment
+
+
+class ControlEnvironment(ShishitoEnvironment):
+    """ Appium control environment. """
+
+    CAPABILITIES = (
+        ('platformName', 'platformName', True, None),
+        ('platformVersion', 'platformVersion', True, None),
+        ('deviceName', 'deviceName', True, None),
+        ('app', 'app', True, None),
+        ('appiumVersion', 'appiumVersion', True, None),
+    )
+
+    def call_browser(self, combination, capabilities=None):
+        """ Starts browser """
+
+        # get browser capabilities and profile
+        if capabilities:
+            capabilities.update(self.get_capabilities(combination))
```

Comment:
it's not good practice modify method arguments. Methods with side-effects are less understandable from outside and can easily lead to mistakes.